_rules/role-required-states-and-properties-4e8ab6.md

```diff
@@ -126,6 +128,28 @@ This `combobox` has the required properties `aria-controls` and `aria-expanded`.
 </ul>
 ```
 
+#### Passed Example 7
```

Comment:
I would update the description of, at least, all the passed examples to also include references to the necessary aria set definition.